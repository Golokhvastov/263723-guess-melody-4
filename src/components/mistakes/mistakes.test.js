import React from "react";
import renderer from "react-test-renderer";
import Mistakes from "./mistakes.jsx";

describe(`GameScreen render correctly`, () => {
  it(`with zero mistakes`, () => {
    const tree = renderer
    .create(
        <Mistakes
          mistakes = {0}
          maxMistakes = {3}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`with on mistake`, () => {
    const tree = renderer
    .create(
        <Mistakes
          mistakes = {1}
          maxMistakes = {3}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
