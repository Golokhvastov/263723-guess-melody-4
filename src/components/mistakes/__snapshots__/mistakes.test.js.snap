// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameScreen render correctly with on mistake 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className="correct"
  />
  <div
    className="correct"
  />
</div>
`;

exports[`GameScreen render correctly with zero mistakes 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="correct"
  />
  <div
    className="correct"
  />
  <div
    className="correct"
  />
</div>
`;
